render.go

```diff
@@ -12,6 +12,7 @@ type Render struct {
 	out                ConsoleWriter
 	prefix             string
 	livePrefixCallback func() (prefix string, useLivePrefix bool)
+	BreakLineCallback  func()
```

Comment:
Done, let me know if you'd like anything else!

As general feedback: one thing that's currently hard with go-prompt is that most members are private, so accessing them is out of the question. There could be things like changing color after the user presses a specific keybind that could be very useful, but are now impossible to do since you can't access those members. My suggestion would be to review the public interface and maybe allow users to be able to access renderer, input parser, etc etc.

Reply:
Thank you for your feedbacks.

We can make the field public without breaking API changes. But we cannot make the public field private without breaking API changes. So in my opinion, I define the field as private if it's not necessary to make public. But as you said, we might need to review the public interface. Thanks!

Reply:
@odino Could you make `BreaklineCallback` private If there is no advantages to make it public in your usecases?

Reply:
Done, let me know if you need anything else!

Reply:
Thank you! LGTM!!
I'll merge this after passes CI checks!